hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/SCMDeletedBlockTransactionStatusManager.java

```diff
@@ -322,9 +324,6 @@ private void removeTimeoutScmCommand(UUID dnId,
           CmdStatusData state = removeScmCommand(dnId, scmCmdId);
           LOG.warn("Remove Timeout SCM BlockDeletionCommand {} for DN {} " +
```

Comment:
I think the log message here could be updated to `LOG.warn("SCM BlockDeletionCommand {} for DN {} was removed after {}ms without update", state, dnId, timeoutMs)`. Let me know what you think?

Reply:
Done.